src/earn/EarnHome.tsx

```diff
@@ -382,4 +390,21 @@ const styles = StyleSheet.create({
     colors: Colors.black,
     marginBottom: Spacing.Thick24,
   },
+  noPoolsContainer: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderColor: 'blue',
```

Comment:
why is this set? in the test plan images I don't see a border

Reply:
Oops, forgot to push my final changes. Thank you!